setup.py

```diff
@@ -0,0 +1,21 @@
+#! /usr/bin/env python
+from setuptools import setup, find_packages
+
+DISTNAME = 'mne-gsoc2018-3d'
```

Comment:
the repo is `mne-gsoc2018-3d` but you can name the module here whatever you want. `mne_gsoc2018_3d` would be the module-name translation, but I'd prefer something shorter like `mne_j3d` (it does not need to match the repo name for now).